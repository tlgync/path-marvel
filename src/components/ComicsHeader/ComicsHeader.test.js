import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { ComicsHeader } from '.';

test('ComicsHeader Render SnapShot', () => {
  const onChange = () => {};
  const placeHolder = 'Search Comics';
  const value = 'spiderman';
  const tree = renderer.create((
    <Router>
      <ComicsHeader onChange={onChange} placeHolder={placeHolder} value={value} />
    </Router>
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
