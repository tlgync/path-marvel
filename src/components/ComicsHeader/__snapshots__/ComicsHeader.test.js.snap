// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComicsHeader Render SnapShot 1`] = `
<div
  className="header"
>
  <div
    className="col-3"
  >
    <input
      onChange={[Function]}
      placeholder="Search Comics"
      value="spiderman"
    />
    <span
      className="focus-bg"
    />
  </div>
  <a
    href="/"
    onClick={[Function]}
  >
    <img
      alt="logo"
      src="../img/logo.svg"
    />
  </a>
</div>
`;
